### PR TITLE
Adminpage

### PR DESCRIPTION
**Pull request by shreyshrey1**
Adding links works and you can click on them within the component
Added "Take Quiz" button, changes to "Review Quiz" based on prop that is passed in. Box turns green and red accordingly.
You can delete lessons based on ID

### CODE REVIEW

File: react_frontend/components/addlesson.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
```
Comment (CodeBrew28) [inline react_frontend/components/addlesson.js]: Don't import * , you dont want to import more then you need

File: react_frontend/pages/AdminPage.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ class AdminPage extends React.Component<Props>{
     render() {
         return (
             <div>I am an admin
-                <LessonForm />
+                <AddLesson style="float:right;"/>
```
Comment (CodeBrew28) [inline react_frontend/pages/AdminPage.js]: Don't inline style, used the styled components to keep it consistent

File: react_frontend/pages/AdminPage.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
```
Comment (CodeBrew28) [inline react_frontend/pages/AdminPage.js]: Same as above, don't import *

File: react_frontend/components/lesson.js [hunk 4]
```diff
@@ -65,38 +66,45 @@ const CheckedBoxInComplete = styled.div`
...
-            <CheckedBoxComplete></CheckedBoxComplete>
-        )
-    }
-    else{
-        return(
-            <CheckedBoxInComplete></CheckedBoxInComplete>
-        )
-    }
-}
+// function isChecked(props) {
```
Comment (CodeBrew28) [inline react_frontend/components/lesson.js]: Delete Comments

File: react_frontend/components/lesson.js [hunk 5]
```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
```
Comment (CodeBrew28) [inline react_frontend/components/lesson.js]: Same as above, dont import *

File: react_frontend/components/addlesson.js [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+    handleChange = (event) => {
+        this.setState({
+            [event.target.name]: event.target.value,
+        });
+    }
+
+    render() {
+        return (
+            <div>
+                <form onSubmit={this.handleSubmit}>Add Lesson:
```
Comment (CodeBrew28) [inline react_frontend/components/addlesson.js]: Seperate the form into it's own component to make it as modular as possible

File: react_frontend/components/lesson.js [hunk 7]
```diff
@@ -65,38 +66,45 @@ const CheckedBoxInComplete = styled.div`
...
     width: 25px;
-    background-color: white;
+    background-color: red;
     z-index: 100;
 `;
 
 
 class LessonComponent extends React.Component<Props>{
     render() {
+        let quiz = null;
```
Comment (CodeBrew28) [inline react_frontend/components/lesson.js]: Keep it DRY (dont repeat yourself), think about how you can not repeat as much code using components here

File: react_frontend/components/addlesson.js [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+                        <input id="lesson_notes_input" name="notes" type="text" value={this.state.notes} onChange={this.handleChange} />
+                    </div>
+
+                    <div>
+                        <label htmlFor="lesson_notes_link_input">Lesson Notes Link: </label>
+                        <input id="lesson_notes_link_input" name="notes_link" type="text" value={this.state.notes_link} onChange={this.handleChange} />
+                    </div>
+
+                    <div>
+                        <label htmlFor="lesson_wksht_input">Lesson Worksheet: </label>
```
Comment (CodeBrew28) [inline react_frontend/components/addlesson.js]: Keep it DRY (dont repeat yourself), think about how you can not repeat as much code using components here

**Comment by shreyshrey1** [comment]
Nooo the navbar...

**Pull request merged**